sdk/search/Microsoft.Azure.Management.Search/tests/Microsoft.Azure.Management.Search.Tests.csproj

```diff
@@ -8,10 +8,19 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <ExcludeFromBuild>true</ExcludeFromBuild>
+    <ExcludeFromTest>true</ExcludeFromTest>
+  </PropertyGroup>
+
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.5.0-preview" />
-    <ProjectReference Include="..\Management.Search\Microsoft.Azure.Management.Search.csproj" />
-    <ProjectReference Include="..\..\DataPlane\Microsoft.Azure.Search\Microsoft.Azure.Search.csproj" />
+    <ProjectReference Include="..\src\Microsoft.Azure.Management.Search.csproj" />
+    <ProjectReference Include="..\..\Microsoft.Azure.Search\src\Microsoft.Azure.Search.csproj" />
```

Comment:
I'm not sure what dependency the management tests have on the data plane. I suspect this reference could be made more specific, maybe to Microsoft.Azure.Search.Common?

Reply:
That would be good to clean-up :)